src/lib/components/Swap/Status/StatusDialog.tsx

```diff
@@ -68,12 +70,18 @@ interface TransactionStatusProps {
 }
 
 function TransactionStatus({ tx, onClose }: TransactionStatusProps) {
+  const { chainId } = useActiveWeb3React()
+
   const Icon = useMemo(() => {
     return tx.receipt?.status ? CheckCircle : Spinner
   }, [tx.receipt?.status])
   const heading = useMemo(() => {
     return tx.receipt?.status ? <Trans>Transaction submitted</Trans> : <Trans>Transaction pending</Trans>
   }, [tx.receipt?.status])
+  const etherscanLink = useMemo(
```

Comment:
I thought you'd made an `Etherscan` component to abstract this away, but I was wrong. I think that'd be a good thing to have, but can be done in another PR.